test/Facade.test.ts

```diff
@@ -1164,6 +1164,18 @@ describe('FacadeRead + FacadeAct contracts', () => {
           },
         ]
       )
+
+      // Nothing should be settleable
+      expect((await facade.auctionsSettleable(rsrTrader.address)).length).to.equal(0)
+
+      // Advance time till auction ended
+      await advanceBlocks(1 + auctionLength / 12)
+
+      // Settleable now
+      expect((await facade.auctionsSettleable(rsrTrader.address)).length).to.equal(0)
```

Comment:
Based on the comment I would expect a different thing here, seems its still not settleable?

Reply:
Very good question, investigating...

Reply:
Mmmm off-by-one-block read vs write difference. Advancing one further block solves the problem.